.github/workflows/certificate-acl.yml

```diff
@@ -0,0 +1,78 @@
+name: Certificate ACL (iam/certificate-acl)
+
+# Triggered after a branch is created, a tag or a branch is pushed or a PR with changes in
+# 'iam/certificat-acl/**' is submitted
+on:
+  create:
+  push:
+  pull_request:
+    paths:
+      - 'iam/certificate-acl/**'
+
+jobs:
+  unit-tests:
+    # Path to help identify each job in the status checks
+    name: (iam/certificate-acl) Unit Tests
+    # If the commit message contains 'ci skip', it is not necessary to run this job
+    if: contains(github.event.head_commit.message, 'ci skip') == false
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+
+    - name: checkout
+      uses: actions/checkout@v1
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: npm install, build, lint and test
+      run: |
+        cd iam/certificate-acl
+        npm ci
+        npm run build --if-present
+        npm run lint
+        npm test
+        npm run check-licenses
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v1
+      with:
+        # Created when 'npm test' was executed
+        file: iam/certificate-acl/coverage/lcov.info
```

Comment:
Try remove `file: iam/certificate-acl/coverage/lcov.info` and change for `directory:iam/certificate-acl/coverage`